### DISCUSSION TITLE
Is it possible to specify a minimum gene number presented in pathways for enrichement?

### DISCUSSION THREAD

**Discussion by fengx90** [egeulgen/pathfindR#118] [Q&A] [answered]
Hello E,

Here I am again. I saw that some enriched pathways have only 2 genes presented. I wonder if it is possible to specify a minimum gene number for the analysis? Or I can only filter the enriched pathways based on the enriched genes in the final table? Thanks very much!

Best,
Feng

**Comment by egeulgen** [comment, marked as answer]
Hey @fengx90,

You can specify a threshold for the minimum threshold of significant genes per active subnetwork. From the documentation:

sig_gene_thr
threshold for the minimum proportion of significant genes in the subnetwork (Default = 0.02) If the number of genes to use as threshold is calculated to be < 2 (e.g. 50 signif. genes x 0.01 = 0.5), the threshold number is set to 2
